### ISSUE TITLE
Task 6.2.3: Configure Automatic Deployment on Main Branch

### ISSUE THREAD

**Issue by mhdsysy** [mhdsysy/appointment-scheduler-for-small-businesses#295]
Set up automatic deployments to production when code is merged into the main branch.

**Comment by mhdsysy** [comment]
**Depends On** [#292](https://github.com/mhdsysy/appointment-scheduler-for-small-businesses/issues/292)
